utils/src/main/scala/com/salesforce/op/utils/spark/OpSparkListener.scala

```diff
@@ -129,6 +143,23 @@ class OpSparkListener
 
 }
 
+trait MetricJsonLike extends JsonLike {
```

Comment:
please add a test for this

Reply:
see https://github.com/salesforce/TransmogrifAI/pull/341/files#diff-f6bf18b1d0ed98372298b89b11dfed41R170